app/Repositories/MessageRepository.php

```diff
@@ -16,7 +16,6 @@ class MessageRepository
     public function create($contest, $data)
     {
         $message = new Message($data);
-        $message->pro_tip = empty($data['pro_tip']) ? null : $data['pro_tip'];
```

Comment:
Just to note, we only really run `create` for Messages during the seeding process and that grabs from the defaults in the `Correspondence` class which is already sets the `pro_tip` to `null`. So no real need to bother with the mutator... at least until we add a `Create Messages` form, which I don't is planned to happen at any point soon 😉